src/js/pre-need/config/schema.json

```diff
@@ -198,23 +198,22 @@
         "street": {
           "type": "string",
           "minLength": 1,
-          "maxLength": 50
+          "maxLength": 35
```

Comment:
These are all copied over from the built JSON schema from `vets-json-schema` as usual for development.